### PR TITLE
Mention dartdoc in style guide

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by abarth** [comment]
LGTM

File: style-guide.md [hunk 1]
```diff
@@ -63,10 +63,67 @@ Example:
 ```
 
 
+Documentation comments (dartdocs)
+---------------------------------
+
+We use "dartdoc" for our documentation. All public members in Flutter
```
Comment (krisgiesing) [inline style-guide.md]: Maybe include a link to https://www.dartlang.org/effective-dart/documentation/#doc-comments?

**Comment by krisgiesing** [comment]
Doh, missed the boat!

File: style-guide.md (hunk 1 above)
Comment (Hixie) [inline style-guide.md]: https://github.com/flutter/website/pull/204

**Pull request merged**